packages/create-sitecore-jss/src/init-runner.ts

```diff
@@ -41,16 +41,19 @@ export const initRunner = async (initializers: string[], args: BaseArgs) => {
 
   await runner(initializers);
 
-  saveConfiguration(args.templates, path.resolve(`${args.destination}${sep}package.json`));
-  // final steps (install, lint, etc)
-  if (!args.noInstall) {
-    installPackages(args.destination, args.silent);
-    lintFix(args.destination, args.silent);
-  }
+  for (const destination of [args.destination, args.nodeAppDestination]) {
+    if (!destination) continue;
+    saveConfiguration(args.templates, path.resolve(`${destination}${sep}package.json`));
+    // final steps (install, lint, etc)
+    if (!args.noInstall) {
+      installPackages(destination, args.silent);
+      lintFix(destination, args.silent);
+    }
 
-  // install pre-push hook if user opts-in
-  if (args.prePushHook) {
-    await installPrePushHook(args.destination, args.silent);
+    // install pre-push hook if user opts-in
+    if (args.prePushHook) {
```

Comment:
Same here (proxy apps don't include pre-push hooks). Skip this for those?

Reply:
Following on my comment above, I added a check to see if install-hook command exists.
I believe the scaffold logic should be flexible and more universal with its flags and logic behind them